cf-promises/cf-promises.c

```diff
@@ -213,12 +215,12 @@ int main(int argc, char *argv[])
 
     if(config->agent_specific.common.show_classes)
```

Comment:
This is no longer a bool, so an explicit comparison to `NULL` would be preferred.